package.json

```diff
@@ -93,6 +93,8 @@
     "styled-components": "^5.2.1",
     "stylelint": "^13.8.0",
     "stylelint-config-recommended": "^3.0.0",
-    "stylelint-config-styled-components": "^0.1.1"
+    "stylelint-config-styled-components": "^0.1.1",
+    "webpack": "^5.10.2",
```

Comment:
Thank you, think this is a bit of fallout from just having swapped to WP5, would have run into this next Monday.

Reply:
no worries, glad add two cents